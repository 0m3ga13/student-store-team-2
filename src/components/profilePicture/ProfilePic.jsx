function ProfilePic() {
    return (
        <div className='mt-5 w-16 h-16 bg-gray-300 rounded-full flex items-center justify-center'>
            <span className='text-gray-600 text-2xl'>A</span>
        </div>
    );
}
export default ProfilePic;
