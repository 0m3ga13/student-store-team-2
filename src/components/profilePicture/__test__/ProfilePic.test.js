import renderer from "react-test-renderer";
import ProfilePic from "../ProfilePic";

it("renders correctly", () => {
    const tree = renderer.create(<ProfilePic />).toJSON();
    expect(tree).toMatchSnapshot();
});
