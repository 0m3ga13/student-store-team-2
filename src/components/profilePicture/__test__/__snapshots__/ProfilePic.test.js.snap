// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-5 w-16 h-16 bg-gray-300 rounded-full flex items-center justify-center"
>
  <span
    className="text-gray-600 text-2xl"
  >
    A
  </span>
</div>
`;
